metadeploy/consumers.py

```diff
@@ -90,6 +90,8 @@ def is_known_model(self, model):
     def has_good_permissions(self, content):
         if self.handle_org_special_case(content):
             return True
+        if self.handle_scratch_org_special_case(content):
+            return True
```

Comment:
@wlonk Do we also want something like this for Preflights and Jobs? Right now, as an anonymous user, I can't subscribe to a Preflight on a Scratch Org I just created.

We should consider this. Do we want to make sure the model is one that an anonymous user actually has access to via their request.session?

Reply:
That seems a likely solution.

Reply:
Replaced by other comments in `models.py`.